_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -273,7 +264,7 @@ function get_initial_state() {
 			'dismissedNotices' => $this->get_dismissed_jetpack_notices(),
 			'isDevVersion' => Jetpack::is_development_version(),
 			'currentVersion' => JETPACK__VERSION,
-			'is_gutenberg_available' => Jetpack_Gutenberg::is_gutenberg_available(),
+			'is_gutenberg_available' => true,
```

Comment:
Is there any benefit to leaving this to call the function?

Reply:
What happens when WP 5.x user runs classic editor plugin?

Reply:
Fair question, same thing as would happen before this PR too. What's the expected outcome?

Reply:
Best I can see, this particular thing only powers the 6.8 Jetpack upgrade notice. We can likely clean that up, but rather land this since it involved wp.com coordination and clean up the admin dash in a separate PR.